src/Internal/ClassHelper.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types=1);
+
+namespace mglaman\PHPStanDrupal\Internal;
+
+use PhpParser\Node\Stmt\Class_;
+
+/**
+ * @internal
+ */
+final class ClassHelper
```

Comment:
Not sure about the naming 😅 

I wonder if we could deal with reflection to use a custom type that could hold those data 🤔
Not sure if it worth the effort right now 🤔

Reply:
Eh, it works. We could name it something like `NamespaceChecker`, since it has methods for checking if the namespace "is" something.

Reply:
I like the `NamespaceCheck` name thanks!